buf.gen.yaml

```diff
@@ -0,0 +1,24 @@
+version: v1
+plugins:
+  - plugin: prost
+    out: bigtable_rs/src/google
+    opt:
+      # - bytes=.
+      - compile_well_known_types
+      - extern_path=.google.protobuf=::pbjson_types
+      - file_descriptor_set
+      - enable_type_names
+      - disable_comments=.
```

Comment:
Would be nice if we can still add the comments and format them in a way without doc-test errors. 

https://github.com/neoeinstein/protoc-gen-prost/issues/112